packages/sommelier.rb

```diff
@@ -2,90 +2,200 @@
-      system "sed -i 's,-I.,-I. -I#{CREW_PREFIX}/include/pixman-1,g' Makefile"
-      system "make PREFIX=#{CREW_PREFIX} SYSCONFDIR=#{CREW_PREFIX}/etc"
-      system "echo 'GDK_BACKEND=x11' > .sommelier.env"
-      system "echo 'CLUTTER_BACKEND=wayland' >> .sommelier.env"
-      system "echo 'XDG_RUNTIME_DIR=/var/run/chrome' >> .sommelier.env"
-      system "echo 'WAYLAND_DISPLAY=wayland-0' >> .sommelier.env"
-      system "echo 'DISPLAY=:0' >> .sommelier.env"
-      system "echo 'SCALE=1' >> .sommelier.env"
-      system "echo '#!/bin/bash' > sommelierd"
-      system "echo 'sommelier -X --x-display=\$DISPLAY --scale=\$SCALE --no-exit-with-child /bin/sh -c \"#{CREW_PREFIX}/etc/sommelierrc\"' >> sommelierd"
-      system "echo '#!/bin/bash' > initsommelier"
-      system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> initsommelier"
-      system "echo 'if [ -z \"\$SOMM\" ]; then' >> initsommelier"
-      system "echo '  [ -f #{CREW_PREFIX}/bin/stopbroadway ] && stopbroadway' >> initsommelier"
-      system "echo '  #{CREW_PREFIX}/sbin/sommelierd &' >> initsommelier"
-      system "echo '  sleep 3' >> initsommelier"
-      system "echo 'fi' >> initsommelier"
-      system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> initsommelier"
-      system "echo 'if [ ! -z \"\$SOMM\" ]; then' >> initsommelier"
-      system "echo '  echo \"sommelier process \$SOMM is running\"' >> initsommelier"
-      system "echo 'else' >> initsommelier"
-      system "echo '  echo \"sommelier failed to start\"' >> initsommelier"
-      system "echo '  exit 1' >> initsommelier"
-      system "echo 'fi' >> initsommelier"
-      system "echo '#!/bin/bash' > stopsommelier"
-      system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> stopsommelier"
-      system "echo 'if [ ! -z \"\$SOMM\" ]; then' >> stopsommelier"
-      system "echo '  killall -g sommelier' >> stopsommelier"
-      system "echo '  if pgrep \"Xwayland\"; then killall Xwayland; fi'"
-      system "echo '  sleep 3' >> stopsommelier"
-      system "echo 'fi' >> stopsommelier"
-      system "echo 'SOMM=\$(pidof sommelier 2> /dev/null)' >> stopsommelier"
-      system "echo 'if [ -z \"\$SOMM\" ]; then' >> stopsommelier"
-      system "echo '  echo \"sommelier process stopped\"' >> stopsommelier"
-      system "echo 'else' >> stopsommelier"
-      system "echo '  echo \"sommelier process \$SOMM is running\"' >> stopsommelier"
-      system "echo '  exit 1' >> stopsommelier"
-      system "echo 'fi' >> stopsommelier"
-      system "echo '#!/bin/bash' > startsommelier"
-      system "echo 'set -a && source ~/.sommelier.env && set +a && initsommelier' >> startsommelier"
+    # There is no good way to checksum the googlesource tgz file, as they appear to be generated on the fly
+    # and checksums vary with each download.
+    system 'curl https://chromium.googlesource.com/chromiumos/platform2/+archive/c6f85a3472584df37fcf2d3d99474081f29256cd.tar.gz | tar mzx --warning=no-timestamp'
```

Comment:
`curl -L` might be smarter here, as redirects may happen, and won't followed without `-L`

Reply:
Good call!